java/test/src/main/java/test/Slice/escape/Key.ice

```diff
@@ -4,71 +4,71 @@
 
 [["java:package:test.Slice.escape"]]
 
-module abstract
+// TODO: figure out a better way to map module names.
+module java_abstract
 {
 
+["java:identifier:_assert"]
```

Comment:
I would pick somewhat recommendable remapping. identifiers with leading underscores or prefixed by java_ are not reasonable recommendations.

Reply:
Typically I agree with this, but here we're talking about the `escape` test.
There isn't a single line of this file that is "reasonable" or "recommendable".

You could argue the whole point of this test is to make sure even the most insane edge-case names still work correctly.

Hopefully the `java_` prefixes are temporary, but we have no way to remap modules, classes, or exceptions right now.
And Slice forbids leading underscores. Sure, we could pick some totally unrelated name, but this would lose the intent.
These are _supposed_ to be tested, we just lack the ability to do so right now.

Reply:
I'd suggest to add an `escaped` for readability, as in `escapedAssert`.

It's weird to have java-prefixed identifiers in Java. Your language is Java, the identifiers are obviously Java identifiers.

Reply:
I updated all the escaped named in Java, C++, and C# to use this new convention.
Our preferred escaping prefix is `escaped` now.

Obviously I'm only talking about places where we can't use a built-in escaping thing like `@` in C#.